From a802464429eab8dc4a802723f52fc52669fc3060 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pali=20Roh=C3=A1r?= <pali.rohar@gmail.com>
Date: Mon, 14 Jan 2013 04:07:23 +0100
Subject: [PATCH 3/3] Rename library kcal_resourceremote to
 kcal_resourceremote_old

* On ubuntu packages depends on one special libkdepim4 version (where is kcal_resourceremote), so install it under another name
---
 kresources/remote/CMakeLists.txt |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/kresources/remote/CMakeLists.txt b/kresources/remote/CMakeLists.txt
index cf7253a..87d95f2 100644
--- a/kresources/remote/CMakeLists.txt
+++ b/kresources/remote/CMakeLists.txt
@@ -7,12 +7,12 @@
 set(kcal_resourceremote_LIB_SRCS resourceremote.cpp resourceremoteconfig.cpp )
 
 
-kde4_add_library(kcal_resourceremote SHARED ${kcal_resourceremote_LIB_SRCS})
+kde4_add_library(kcal_resourceremote_old SHARED ${kcal_resourceremote_LIB_SRCS})
 
-target_link_libraries(kcal_resourceremote  ${KDE4_KDECORE_LIBS} ${KDEPIMLIBS_KCAL_LIBS} kdepim_old )
+target_link_libraries(kcal_resourceremote_old  ${KDE4_KDECORE_LIBS} ${KDEPIMLIBS_KCAL_LIBS} kdepim_old )
 
-set_target_properties(kcal_resourceremote PROPERTIES VERSION ${GENERIC_LIB_VERSION} SOVERSION ${GENERIC_LIB_SOVERSION} )
-install(TARGETS kcal_resourceremote ${INSTALL_TARGETS_DEFAULT_ARGS})
+set_target_properties(kcal_resourceremote_old PROPERTIES VERSION ${GENERIC_LIB_VERSION} SOVERSION ${GENERIC_LIB_SOVERSION} )
+install(TARGETS kcal_resourceremote_old ${INSTALL_TARGETS_DEFAULT_ARGS})
 
 
 ########### next target ###############
@@ -29,7 +29,7 @@ target_link_libraries(kcal_remote
   ${KDE4_KDECORE_LIBS}
   ${KDEPIMLIBS_KCAL_LIBS}
   ${KDEPIMLIBS_KRESOURCES_LIBS}
-  kcal_resourceremote
+  kcal_resourceremote_old
   kdepim_old
 )
 
-- 
1.7.10.4

